backend/api/Controllers/MissionController.cs

```diff
@@ -234,6 +234,51 @@ public async Task<ActionResult<byte[]>> GetMap([FromRoute] string id)
         return CreatedAtAction(nameof(GetMissionById), new { id = newMission.Id }, newMission);
     }
 
+    /// <summary>
+    /// Schedule a custom mission
+    /// </summary>
+    /// <remarks>
+    /// <para> This query schedules a custom mission defined in the incoming json </para>
+    /// </remarks>
+    [HttpPost]
+    [Route("custom")]
+    [ProducesResponseType(typeof(Mission), StatusCodes.Status201Created)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<Mission>> Create(
+        [FromBody] CustomMissionQuery customMissionQuery
+    )
+    {
+        var robot = await _robotService.ReadById(customMissionQuery.RobotId);
+        if (robot is null)
+            return NotFound($"Could not find robot with id {customMissionQuery.RobotId}");
```

Comment:
Should a 404 response be included as `ProducesResponseType`?

Reply:
Yes, good catch!